lessons/Data-Science/AI/ai-training/langs/en-us/pages/human-judgment.adoc

```diff
@@ -0,0 +1,69 @@
+= The Importance of Human Judgment
+
+Open the @starter-file{plagiarism}, click "Run", and then respond to the questions below.
+
+== Determine Closeness
+
+@n Use `cosine-similarity` to compare the student-submitted essay with the wikipedia article. Click "Run". What does the program tell you about the comparison? @fitb{}{@ifsoln{`cosine-similarity(student-essay, wiki-article)` produces an output of ~0.97087...,}}
+
+@fitb{}{@ifsoln{ indicating that the texts are extremely close.}}
+
+@n Did you know that "pachyderm" is a synonym for "elephant"? In the student essay, replace all instances of "elephant" with "pachyderm". Try running the program again, to compare `student-essay` and `wiki-article`. What does the program tell you about the comparison?
```

Comment:
we could easily also predefine this in the starter file and tell them that the student who wrote `student-essay-2` did  this swap and then have them run it.